stats/thresholds.go

```diff
@@ -213,7 +216,21 @@ func (ts Thresholds) MarshalJSON() ([]byte, error) {
 		configs[i].AbortOnFail = t.AbortOnFail
 		configs[i].AbortGracePeriod = t.AbortGracePeriod
 	}
-	return json.Marshal(configs)
+
+	return MarshalJSONWithoutHTMLEscape(configs)
+}
+
+// MarshalJSONWithoutHTMLEscape marshals t to JSON without escaping characters
+// for safe use in HTML.
+func MarshalJSONWithoutHTMLEscape(t interface{}) ([]byte, error) {
```

Comment:
I would've liked to move this to a helpers file in `lib`, and also reuse it here: https://github.com/k6io/k6/blob/af1e032ebf949cde092d4a2eddb78528ea0a5864/lib/archive.go#L384-L395

But unfortunately that creates an import cycle (`stats`->`lib`->`stats`), so I left it here.